substrate/client/network/src/litep2p/shim/notification/config.rs

```diff
@@ -108,6 +108,7 @@ impl NotificationProtocolConfig {
 			.with_handshake(handshake.map_or(vec![1], |handshake| (*handshake).to_vec()))
 			.with_max_size(max_notification_size as usize)
 			.with_auto_accept_inbound(true)
+			.with_sync_channel_size(20 * 2048)
```

Comment:
How as this number chosen ? cc @lexnv

Reply:
Why aren't we using `SYNC_NOTIFICATIONS_BUFFER_SIZE` const ?

Reply:
reverted the change